svg/elements/feGaussianBlur.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/feGaussianBlur",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
I've used http://staging-dot-mdn-bcd-collector.appspot.com/tests/api/SVGFEGaussianBlurElement to confirm support back to Chrome 15, so this is correct. It's also supported on current Chrome for Android, so I'll set that to true as well.